workers/unity/Packages/com.improbable.gdk.ui/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "com.improbable.gdk.ui",
+    "displayName": "SpatialOS Editor UI",
+    "version": "0.0.1",
```

Comment:
Probably want to make the name of the package - `com.improbable.gdk.editorui` or similar. Will differentiate between a "runtime UI feature module"